src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+$c-grey: #eee;
+$squareSize: 100px;
+$padding: 10px;
+$gap: 1px;
+$week: 7;
+$weekStartMON: 0;
+$dayNamesLeft: (
+  "mon": 1,
+  "tue": 2,
+  "wed": 3,
+  "thu": 4,
+  "fri": 5,
+  "sat": 6,
+  "sun": 7
+);
+
+%flex-center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+
+  height: 100vh;
+  @extend %flex-center;
+}
```

Comment:
I think these lines you need to put into the new doc: styles/utils/_normalize.scss or _reset.scss

Reply:
![image](https://user-images.githubusercontent.com/120763734/216832583-b822f695-85d8-4683-8cff-98aa60e78f80.png)